pkg/sdkclient/grpc_resolver/client_resolver.go

```diff
@@ -81,12 +79,15 @@ func (*multiProcResolver) ResolveNow(o resolver.ResolveNowOptions) {}
 func (*multiProcResolver) Close()                                  {}
 func (*multiProcResolver) Resolve(target resolver.Target)          {}
 
-// BuildConnAddrs Populate the connection list for the clients
-// Format (serverAddr, serverIdx) : (0.0.0.0:5551, 1)
-func BuildConnAddrs(numCpu int) []string {
+// buildConnAddrs Populate the connection list for the clients
+// Format (serverAddr, serverIdx) : (0.0.0.0:5551, 1), (0.0.0.0:5552, 2)
+func buildConnAddrs(numCpu int, servPorts []string) []string {
```

Comment:
`numCpu` could be potentially not equal to `len(servPorts)`, so the logic below is risky.

Reply:
From the SDK side they will both be the same value as numCpu is written to the server info as the number of server instances required in multiproc. 
But I can actually do away with the variable itself and just use servPorts and len(servPorts), I guess would be easier for understanding